skymp5-server/cpp/server_guest_lib/PacketParser.cpp

```diff
@@ -262,6 +270,8 @@ void PacketParser::TransformPacketIntoAction(Networking::UserId userId,
       break;
     }
     default:
-      throw PublicError("Unknown MsgType: " + std::to_string((TypeInt)type));
+      simdjson::dom::element data_;
+      ReadEx(jMessage, JsonPointers::data, &data_);
+      actionListener.OnUnknown(rawMsgData, data_);
```

Comment:
I think the whole object should be printed, not just data. (Probably with some generic validation and truncating so we won't spam console with huge messages with like 1K bytes or more)